features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/asset_api_router.jag

```diff
@@ -1121,10 +1123,9 @@ require('/modules/publisher.js').exec(function(ctx) {
             print(msg);
 
         } catch (e) {
-            log.debug(e);
+            log.error(e);
+			log.error('The asset ' + id + ' could not be updated.');
```

Comment:
Fixed (for line [988](https://github.com/wso2/carbon-appmgt/pull/77/files#diff-c3e3136fda7df54a282fdce4b2171534R988), & [1126](https://github.com/wso2/carbon-appmgt/pull/77/files#diff-c3e3136fda7df54a282fdce4b2171534R1126)) in commit fceee5a9dbf8eba5e3f3a0ed56e01ec1825bcfb7 of PR #248